haproxy/state.go

```diff
@@ -79,6 +79,16 @@ func (h *HAProxy) watch(sd *lib.Shutdown) error {
 				waitAndRetry()
 				continue
 			}
+			// Don't check spoe filter index since it is incremented on dataplane api because retrieved using filters() routine
```

Comment:
Isn't there any way to have a consistent index? Is this behavior expected?

Reply:
The API doesn't offer this https://www.haproxy.com/documentation/dataplaneapi/community/#get-/services/haproxy/configuration/filters - you can create only one filter at a time